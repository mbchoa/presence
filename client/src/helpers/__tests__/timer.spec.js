import calculateOverflowTime from '../timer.js';

describe('timer test suite', () => {
  it('should return correct number of seconds', () => {
    expect(calculateOverflowTime(3500)).toMatchSnapshot();
  });

  it('should return correct number of minutes', () => {
    expect(calculateOverflowTime(125000)).toMatchSnapshot();
  });

  it('should return correct number of hours', () => {
    expect(calculateOverflowTime(3652341)).toMatchSnapshot();
  });
});
