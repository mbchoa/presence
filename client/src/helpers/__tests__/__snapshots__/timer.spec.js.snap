// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timer test suite should return correct number of hours 1`] = `
Object {
  "hours": 1,
  "minutes": 0,
  "ms": 34,
  "seconds": 52,
}
`;

exports[`timer test suite should return correct number of minutes 1`] = `
Object {
  "hours": 0,
  "minutes": 2,
  "ms": 0,
  "seconds": 5,
}
`;

exports[`timer test suite should return correct number of seconds 1`] = `
Object {
  "hours": 0,
  "minutes": 0,
  "ms": 50,
  "seconds": 3,
}
`;
